### DISCUSSION TITLE
"Data source" issues. Problem with getting nested tags to work and no properties support as "Data source"?

### DISCUSSION THREAD

**Discussion by TheMastish** [marcusolsson/obsidian-projects#673] [Help] [answered]
# Issue:
I'm trying to integrate this plugin into my workflow, but I can't seem to get [nested tags](https://help.obsidian.md/Editing+and+formatting/Tags#Nested+tags) to work.

# Goal

My goal is beeing able to create project hubs using this plugin that will be based on category and subcategory using:

- #p
- #p/💼Category
- #p/💼Category/🍬Subcategory

that will also let me menage all things related to the project, so:

- #p/💼Category/🍬Subcategory/[Meeting/Note/Task]

Newly created files have to be added automatically to the corresponding Project plugin "project" besed on nested tag. 

# My file structure

I put all my projects into diffecent catgory and subcategory folders (here is total of 4 main categories and each have between 4-15 subcategories). Whenever i create a new project it creates "Project" folder in selected subcategory like this:

```
.
└──💼Category/
    └── 🍬Subcategory/
        └── Subcategory projects/
 here->    └── Project/
                ├── Project meetings
                │   └── meeting.md
                ├── Project notes
                │   └── note.md
                ├── Project attachments
                │   └── attachment.xlsx
                ├── Project tasks
                │   ├── kanban.md
                │   └── task.md
                └── Project note.md
```

Frontmatter of "project note.md" looks like this:

```
fileClass: Projekt
p_category: 💼Category
p_subCategory: 🍬Subcategory
tags:
- p/💼Category/🍬Subcategory
```

I also have meetings, notes and tasks that are automatically created for selected project using Quickadd and Templater in it's parent folder locations. "Project note.md" acts like a central note with the whole project history and all "[Meeting/Note/Task]" are linked to this file using Obsidian v1.4 linking in property

Meeting/Note/Task front matter looks like this:

```
fileClass: [Meeting/Note/Task]
parent: "[[Project note.md]]"
p_category: 💼Category
p_subCategory: 🍬Subcategory
tags:
- p/💼Category/🍬Subcategory/[Meeting/Note/Task]
```

# What i've tried

- Using folders. In my case it's a no-go because it requires too much manual work every time now category is created
- Using Dataview. It doesn't allow me to interact with the files, and is "View only".
- Using nested tags - it doesn't seem to work however i try to set this up. tried:
#p
#p/
#p*
#p/*

In theory I could rewrite all my tagging logic, but it would require me to rewrite a ton of nested tags into individual tags, but no "Tag wrangler" would help me with that.

Is there a way to use nested tags, and i'm just doing something wrong. Maybe I can one use properties as data source other than using dataview? Perhaps emojis fcks with the display?

**Comment by Acylation** [comment, marked as answer]
Thanks for raising this. Nested tag support is now in progress, please wait for #650 being merged. Tag data source is built for your case and there are several improvements on the road.
